packages/experimental/AppearanceAdditions/ios/FRNAppearanceAdditions.m

```diff
@@ -137,9 +136,21 @@ - (void)stopObserving {
 
 - (void)appearanceChanged:(NSNotification *)notification {
     if (_hasListeners) {
+        RCTBridge *currentBridge = [self bridge];
+        RCTBridge *notificationBridge = [[notification object] bridge];
+        
+        if (![notificationBridge isKindOfClass:[RCTBridge class]]) {
+            return;
+        }
+        
+        // Don't send the appearanceChanged event if the notification didn't originate from the same react native instance
+        if ([currentBridge bundleURL] != [notificationBridge bundleURL]) {
```

Comment:
Same as earlier: should be able to just check literal object equality.

Reply:
Updated to check is the pointers to the bridges refer to the same instance